manager/src/grype_db_manager/cli/listing.py

```diff
@@ -19,7 +19,7 @@ def group(_: config.Application) -> None:
 @click.option("--ignore-missing-listing", "-i", default=False, help="ignore missing listing from S3", is_flag=True)
 @click.pass_obj
 @error.handle_exception(handle=(ValueError,))
-def create_listing(cfg: config.Application, ignore_missing_listing: bool) -> str:
+def create_listing(cfg: config.Application, ignore_missing_listing: bool) -> None:
```

Comment:
An earlier incarnation of this added a `listing_file` argument, but this ended up reading stuff from S3 twice, so I thought it would be better to just consolidate the building of the listing files so this happens once.